Tetrahedral_remeshing/include/CGAL/Tetrahedral_remeshing/Remeshing_cell_base_3.h

```diff
@@ -50,26 +50,7 @@ It has the default value `Triangulation_cell_base_3<Gt>`.
 */
 template<typename Gt,
          typename Cb = CGAL::Triangulation_cell_base_3<Gt> >
-class Remeshing_cell_base_3
-#ifndef DOXYGEN_RUNNING
-  : public CGAL::Mesh_cell_base_3<Gt, internal::Fake_MD_C, Cb>
-#endif
-{
-  typedef CGAL::Mesh_cell_base_3<Gt, internal::Fake_MD_C, Cb> Base;
-  typedef typename Base::Vertex_handle Vertex_handle;
-  typedef typename Base::Cell_handle   Cell_handle;
-
-public:
-  // To get correct cell type in TDS
-  template < class TDS2 >
-  struct Rebind_TDS
-  {
-    typedef typename Cb::template Rebind_TDS<TDS2>::Other Cb2;
-    typedef Remeshing_cell_base_3<Gt, Cb2> Other;
-  };
-
-  using Base::Base;
-};
+using Remeshing_cell_base_3 = CGAL::Mesh_cell_base_3<Gt, internal::Fake_MD_C, Cb>;
```

Comment:
I would like to discuss that with @janetournois. And in particular, what to do for the documentation.

Reply:
@janetournois I do not remember. Have we decided something about the documentation?

Reply:
No, but this version should not need extra documentation, does it?

Reply:
The current version expose the alias templates as they are:  
https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-5.1-Ic-137/output2/Tetrahedral_remeshing/group__PkgTetrahedralRemeshingClasses.html

Is that what we want?